### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kat10140**
**Related Issue:** #

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,13 +15,16 @@ A collection of calcite components for building single page applications.
 - [calcite-action-bar](/src/components/calcite-action-bar/readme.md)
 - [calcite-action-pad](/src/components/calcite-action-pad/readme.md)
 - [calcite-block](/src/components/calcite-block/readme.md)
+- [calcite-filter](/src/components/calcite-filter/readme.md)
```
Comment (pr3tori4n) [inline README.md]: I believe we wanted to hold off on advertising the filter as a public component for now. @driskull @jcfranco that right?

**Comment by kat10140** [comment]
should be all good @pr3tori4n

**Comment by kat10140** [comment]
@pr3tori4n is this fine now? i can't merge until you approve

**Comment by kat10140** [comment]
@pr3tori4n pls approve

**Pull request merged**